common/src/main/java/com/discordsrv/common/config/documentation/DocumentationConfig.java

```diff
@@ -0,0 +1,5 @@
+package com.discordsrv.common.config.documentation;
+
+public class DocumentationConfig {
```

Comment:
I would prefer if this class was called `DocumentationUrls` or something similar as this isn't a config